ASIM/dev/ASimTester/filteringTest/ASimFilteringTest.py

```diff
@@ -218,6 +273,105 @@ def scalar_test_check_fictive_value(self, param_name, query_definition, column_n
             self.assertEqual(0, len(no_results_response.tables[0].rows), f"Parameter: {param_name} - Returned results for non existing filter value. Filtered by value: {DUMMY_VALUE}")
 
         
+    # Returning an array of at most two values from the query response, each string in the returned array is not contained in the other
```

Comment:
this comment is not clear, what is the meaning of "each string in the returned array is not contained in the other"

Reply:
Changed the comment.